noobaa/noobaa-core
# Remote syslog to new architecture

### Explain the changes:
- Remote syslog to new architecture

### Issues: Fixed / Gap #xxx
- 

### Testing Instructions:

1. go to management
2. expand Remote Syslog
3. switch on remote syslog
4. type address '1.1.1.1'
5. dirty mark is shown near the header
6. click Apply Changes
7. data updated successfully
8. success notification was shown


### Reminders:
- Create a new test - the first is the hardest
- User Experience is top priority
- Design for failure
- Keep it simple
- `npm test`
- Imagine a support call - Add diagnostics info, phonehome info, activity log events, external syslog
- Upgrade - DB schema, server platform, agents install, npm packages
